libraries/AP_Camera/AP_Camera.cpp

```diff
@@ -32,6 +32,13 @@ const AP_Param::GroupInfo AP_Camera::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("_AUTO_ONLY",  10, AP_Camera, _auto_mode_only, 0),
 
+    // @Param: _OLD_FDBK
+    // @DisplayName: Send deprecated feedback message
+    // @Description: When enabled, a successful image capture will send the deprecated CAMERA_FEEDBACK message as well as the CAMERA_IMAGE_CAPTURED message
+    // @Values: 0:Disabled,1:Enabled
+    // @User: Standard
+    AP_GROUPINFO("_OLD_FDBK",  11, AP_Camera, _use_deprecated_feedback_msg, 0),
```

Comment:
.... actually, we proably want a `CAMERA_OPTIONS` bitmask instead... and a AP::camera().option_is_enabled(Camera::Option::FEEDBACK_MESSAGE)`

Reply:
Fixed.